examples/sigstore-go-verification/README.md

```diff
@@ -1,9 +1,51 @@
+# Building examples
+
+To build example programs, run `make build-examples` in the repo root.
+The built executables will be in the `examples/` subdirectory:
+```shell
+$ make build-examples && ls -tr examples | tail -3
+go build -C ./examples/oci-image-verification -o oci-image-verification .
+go build -C ./examples/sigstore-go-signing -o sigstore-go-signing .
+go build -C ./examples/sigstore-go-verification -o sigstore-go-verification .
+
+$ find examples -type f -perm -u+x | sort
+examples/oci-image-verification/oci-image-verification
+examples/sigstore-go-signing/sigstore-go-signing
+examples/sigstore-go-verification/sigstore-go-verification
+```
+
+# oci-image-verification
+
+This is a CLI fo verifying signatures on the OCI images. View the help text with `-h` or `--help` for all the options.
+(The usage example below is not intended to represent the best practices - add all the restrictions and verification parameters
+such as `-expectedIssuer` and `-expectedSAN` applicable to your environment.)
+```shell
+./oci-image-verification \
+  -requireTlog=false -ignore-sct -expectedIssuerRegex='.*' -expectedSANRegex='.*' \
+  -trustedrootJSONpath=$HOME/dev/files/trustedroot.json -ociImage docker.company.com:4443/repo/image/name
+```
```

Comment:
I wonder if we should just have the output of `./oci-image-verification -h` here, like we do for `sigstore-go-signing` below.

The example for `./sigstore-go-verification` makes more sense, because we distribute the files you need to run the example, but that probably won't be the case for `./oci-image-verification`.